gnmi_server/clientCertAuth.go

```diff
@@ -44,9 +127,92 @@ func ClientCertAuthenAndAuthor(ctx context.Context, serviceConfigTableName strin
 		}
 	}
 
+	if enableCrl {
+		err := VerifyCertCrl(tlsAuth.State)
+		if err != nil {
+			glog.Infof("[%s] Failed to verify cert with CRL; %v", rc.ID, err)
+			return ctx, err
+		}
+	}
+
 	return ctx, nil
 }
 
+func TryDownload(url string) bool {
```

Comment:
If we need to download crl from external url, we need make sure dns is working.

Reply:
There is no external url, I verified the dns is working.